homeassistant/components/upnp/translations/en.json

```diff
@@ -1,26 +1,25 @@
 {
     "config": {
+        "flow_title": "{name} ({host})",
```

Comment:
You need to update `strings.json`, not `translations/en.json`. Also, for steps where the `title` is the same as the name in the manifest, you can omit it and it will handle it appropriately.

Reply:
Updated `strings.json`. Do you want me to revert `translations/en.json`?

Reply:
nah, they will be overridden with Lokalise data the first UTC midnight after merge :)